package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "my-app",
+  "name": "giphy-search-app",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@chakra-ui/react": "^1.8.7",
+    "@chakra-ui/react": "^2.3.5",
     "@emotion/react": "^11.0.0",
     "@emotion/styled": "^11.0.0",
+    "eslint-config-airbnb": "^19.0.4",
     "framer-motion": "^4.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
Why do we need this package?

Reply:
It wasn't completely necessary but I bumped react up to version 18 so that I could use version 2 of the Chakra UI library. I also thought for a second that we didn't need react-dom but come to find out even though react v18 uses `createRoot` instead of `render,` `createRoot` is still found in react-dom so, best I can tell, we still need react-dom.